### PR TITLE
feat: update loading icon

### PR DESCRIPTION
**Pull request by ianlapham**
- Used loading icon from app with updated styles to match widget theme

### CODE REVIEW

File: src/lib/icons/index.tsx [hunk 1]
```diff
@@ -133,3 +133,48 @@ export const Spinner = styled(icon(SpinnerIcon))<{ color?: Color }>`
   stroke-linecap: round;
   stroke-width: 2;
 `
+
```
Comment (zzmp) [inline src/lib/icons/index.tsx:136]: This is long enough that I think it should go into its own file, src/lib/icons/DetailedSpinner.
Also, everything exported from this file is an Icon. Do the types work in Caption, or does the definition of Icon need to be expanded?

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/lib/icons/index.tsx (hunk 1 above)
Comment (ianlapham) [reply src/lib/icons/index.tsx:136]: types actually work fine in Caption no changes needed, but yeah can move this into its own file.

File: src/lib/icons/DetailedSpinner.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+to {
+  transform: rotate(360deg);
+}
+`
+
+const DetailedSpinnerWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  height: 1rem;
```
Comment (zzmp) [inline src/lib/icons/DetailedSpinner.tsx:15]: Considering that the InnerDot and OuterRing are defined in terms of px, I think the height and width should be set to 16px and not 1rem.

File: src/lib/icons/DetailedSpinner.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import styled, { keyframes } from 'lib/theme'
```
Comment (zzmp) [inline src/lib/icons/DetailedSpinner.tsx:1]: nit: this spinner is statically sized to 16px, so I think it should be renamed InlineSpinner.

**Review by zzmp — APPROVED** [review]
two quick changes, but lgtm, no re-review needed

File: src/lib/icons/DetailedSpinner.tsx (hunk 3 above)
Comment (ianlapham) [reply src/lib/icons/DetailedSpinner.tsx:1]: Good call

**Pull request merged**